docs/api/qiskit/release-notes/0.11.md

```diff
@@ -62,7 +62,7 @@ IBMQ.update_account()
 For more complex cases or fine tuning your configuration, the following methods are available:
 
 *   the `IBMQ.delete_accounts()` can be used for resetting your configuration file.
-*   the `IBMQ.save_account('MY_TOKEN')` method can be used for saving your credentials, following the instructions in the [IBM Q Experience v2](https://quantum-computing.ibm.com) account page.
```

Comment:
Bad

Reply:
This is just the original? Doesn't start with `docs.` so it is as it was, what should this be in that case? It's just `quantum-computing` without any subdomain on it

Reply:
It shouldn't have any change at all in this PR because this PR is only about changing to relative links when possible, not changing `quantum-computing` to `quantum`. That's so that we can land your PR right now, rather than needing to wait until the URL changes. Does that make sense? 

This applies to most links in this PR.